src/css/ContactLinks.scss

```diff
@@ -1,29 +1,32 @@
 /*------------------------------------*\
   #ContactLinks
 \*------------------------------------*/
-$iconBackgroundColor:#4F7DF3;
+$iconColor:#4F7DF3;
 $copyWriteTextColor:#969696;
 
 *{
```

Comment:
always have a single space between the selector and bracket. Fix this issue throughout the entire file.

Reply:
Could you tell me what line number you are referring to?

Reply:
On **line 7 and throughout this entire file**, there should be a space between the selector and bracket `* {` not `*{`